VMAccess/vmaccess.py

```diff
@@ -67,6 +67,10 @@ def install():
 def enable():
     hutil = Util.HandlerUtility(waagent.Log, waagent.Error)
     hutil.do_parse_context('Enable')
+
+    if hutil.get_extension_version() == "1.4.5.0":
+        _forcefully_reset_chap()
```

Comment:
This should be a one time thing to correct our previous mistake.  I wanted to contain it to this hotfix, and then remove it later...good point though in that this does not help us if we re-publish hotfixes for the other available versions (1.0, 1.1, 1.2, and 1.3).  I'll remove it.

I always miss that I have to update manifest.xml and CHANGELOG.md.  Will do.